charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 All changes to this chart will be documented in this file.
 
 ## [TBD]
-* Fix PDB typo
+
+* `sonarqube-dce`-only: Introduce `ApplicationNodes.podDistributionBudget` and `searchNodes.podDistributionBudget` in place of deprecated `ApplicationNodes.podDistributionBudget` and `searchNodes.podDistributionBudget`.
```

Comment:
this does not require to be prefixed by `sonarqube-dce-only` as it is already in the `sonarqube-dce` chart folder.